src/components/layout/Header.vue

```diff
@@ -1,12 +1,41 @@
 
 <template>
-  <header class="banner">
+  <header class="banner" style="position:relative">
     <h1>Attendance Tracker</h1>
+    <div v-if="isAuthenticated" v-on:click="$router.push('home')" class="profileButton">  <!--Only shows after login-->
```

Comment:
The router push takes a path so you either need to give it '/home' or {name:'home'} for it to work properly.
```suggestion
    <div v-if="isAuthenticated" v-on:click="$router.push({name:'home'})" class="profileButton">  <!--Only shows after login-->
```

Reply:
That was the style used in the Login and Dashboard buttons. It worked fine for those and also when I tested this.

Reply:
The login and dashboard use the {name:'nameOfPage'} as I have suggested. Only 'home' doesn't work since the router interprets it as add 'home' to the end of the current URL which is incorrect.

Reply:
LoginForm.vue Line 34:
`<button type="button" @click="$router.push('create-account')">Create Account</button>`

Reply:
Ah yeah I didn't notice that one. So this one just happens to work since it only happens on the home page (/home) which means if the router replaces the last part of the URL with 'create-account' it works (/home becomes /create-account). In general this wouldn't work from any page (my mistake, that's one of the first ones I had done and I didn't notice it since it works 😄)